client/pages/result.jsx

```diff
@@ -128,6 +128,7 @@ export default class Result extends React.Component {
             <MapsComponent maps={this.state.maps} />
           </div>
           <Accordion reviews={this.state.reviews} />
+          <div className='height-13vh'></div>
```

Comment:
This is what margin is for try that instead.

Reply:
![image](https://user-images.githubusercontent.com/90667339/156830134-a9a6bc2c-adf5-496d-ab8f-26c30c5c5dfd.png)